doc/source/conf.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/stable/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'gcovr'
+copyright = u'2018, William Hart, John Siirola and and Lukas Atkinson.'
+author = u'William Hart, John Siirola and and Lukas Atkinson.'
+
+# The short X.Y version
+version = u''
+# The full version, including alpha/beta/rc tags
+release = u''
```

Comment:
We could get htese information from the last git tag on this branch using `git describe`:

~~~
#to get the git revision (see http://dreamiurg.net/2011/10/03/using-git-to-get-version-information/)
from subprocess import Popen, PIPE

def get_version():
    #"""
    #Returns project version as string from 'git describe' command.
    #"""
    pipe = Popen('git describe --tags --always', stdout=PIPE, shell=True)
    version = pipe.stdout.read()

    if version:
        return version.decode('utf-8')
    else:
        return 'X.Y'

version = get_version().rstrip()
release = version
~~~

Is this ok?

Reply:
Why not use the same kind of thing as in `setup.py`:
https://github.com/gcovr/gcovr/blob/869bbc8d68c00499c1a374045f79f17f77192330/setup.py#L35

Reply:
This should work. But using `./gcovr/version.py` would always report `3.4` for every version after 3.4. `git describe --tags --always` will report something like `3.4-65-g869bbc8` if you are not exactly on a tag. So you see, that it is an intermediate version.

Reply:
Thank you for everyone's suggestions!

Right now, the guide is only published manually for a release, so that the published guide always refers to the most recent released version.
In the long run, I'd like to automate this so that we can publish the docs for both the last release and the current development version (via readthedocs?). In either case, I'm not sure that tying a documentation build to a git commit is valuable.

If there's no simple way to include a version, I'd rather keep this blank for now and revisit this if and when giving a clear version would be necessary. E.g. I think readthedocs wouldn't use the project version from the sphinx config at all.